README.md

```diff
@@ -68,6 +68,20 @@ You can configure the verbosity with these flags :
 
 ## Deployment
 The official documentation is available here : [website](https://book.getfoundry.sh/reference/forge/deploy-commands) 
+### Script
+To run the script for deployment, you need to create a .env file. The value for CMTAT.ADDRESS is require only to use the script **RuleEngine.s.sol**
+Warning : put your private key in a .env file is not the best secure way.
```

Comment:
-> 'Warning: if the private key is in a .env file, the file should be protected from unuautorized access (for example, with permission `600` only).'